### PR TITLE
UltraISO: Add version 9.7.1.3519

### PR DESCRIPTION
**Pull request by issaclin32**
**UltraISO** ([homepage](https://www.ezbsystems.com/ultraiso/)) is a CD/DVD image file editing tool.

Notes:
* **UltraISO** is not a free software, but users can still create and modify CD images (under 300 MB) in trial mode.

Please let me know if there are any problems. Thanks.

### CODE REVIEW

File: bucket/ultraiso.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://www.ezbsystems.com/ultraiso/",
+    "description": "CD/DVD image file editing tool",
+    "version": "9.7.1.3519",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://www.ezbsystems.com/pad/ultraiso.xml"
+    },
+    "url": "https://www.ezbsystems.com/dl1.php?file=uiso9_pe.exe",
```
Comment (Ash258) [inline bucket/ultraiso.json]: This url leads to http 

A

**Comment by rasa** [comment]
Perhaps we move trial software to a separate bucket, say `nonfree` or `paid`?

**Comment by rasa** [comment]
I lack the permissions to create a repo under [ScoopInstaller](https://github.com/ScoopInstaller). If someone creates it, I will move this package there.

**Comment by issaclin32** [comment]
Related discussion: https://github.com/lukesampson/scoop/issues/3828

**Comment by issaclin32** [comment]
I am closing this PR because it is outdated.

I will open a new PR when *Scoop* allows **non-free** or **trial** programs. (probably in a new bucket other than `extras`)

**Pull request closed without merging**